airbyte-integrations/connectors/source-always-works/docker-shim-mvp/destination-listen-and-echo/listen-and-echo.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+echo "ip: ${DEST_IP}", "port: ${DEST_PORT}"
```

Comment:
these are env var, since I imagine we when call this from Kube it's easier to set env vars.